common/lib/xmodule/xmodule/x_module.py

```diff
@@ -72,7 +72,9 @@
 # Views that present a "preview" view of an xblock (as opposed to an editing view).
 PREVIEW_VIEWS = [STUDENT_VIEW, PUBLIC_VIEW, AUTHOR_VIEW]
 
-DEFAULT_PUBLIC_VIEW_MESSAGE = u'Please enroll to view this content.'
+DEFAULT_PUBLIC_VIEW_MESSAGE = u'''
+This content is only accessible to enrolled learners. Sign in or register, and enroll in this course to view it.
+'''
```

Comment:
Nit: I'm not sure if it affects the translation to have a starting and trailing newline. Could you please use multiple strings inside a paren block to concatenate it without the newlines, like you do for `display_text` later on in this PR)? So something like:

```python
DEFAULT_PUBLIC_VIEW_MESSAGE = (
    u'This content is only accessible to enrolled learners. '
    u'Sign in or register, and enroll in this course to view it.'
)
```

Reply:
I've modified it to use multiple strings.